pkg/plugins/plugins.go

```diff
@@ -144,26 +144,8 @@ func ReadPluginJSON(reader io.Reader) (JSONData, error) {
 		return JSONData{}, err
 	}
 
-	// Hardcoded changes
-	switch plugin.ID {
-	case "grafana-piechart-panel":
+	if plugin.ID == "grafana-piechart-panel" {
```

Comment:
need this for https://github.com/grafana/grafana/pull/95955/files#diff-6cb7474613e71735ab205b028b5b52a5deb39fc1ddfdd4101402e0e313ac70cbR108